lower_bound: [30, 0, 0]
upper_bound: [30, 255, 255]
box_offset: 0
enhanced_dim: [200, 400]
value_backside: 1000000
crop_left_right: 25
crop_top_bottom: 50
classes: {0: 'back', 1: 'green', 2: 'red', 3: 'yellow'}
nn_input_size: (None, 64, 64, 3)
weights_path: 'traffic_light_detection/weights.h5'